docs/speech-to-text/features/speaker-identification.mdx

```diff
@@ -0,0 +1,60 @@
+---
+description: "Learn how Speechmatics identifies speakers in audio"
+keywords:
+  [
+    speechmatics,
+    features,
+    speaker identification,
+    diarization,
+    transcription,
+    speech recognition,
+    automatic speech recognition,
+    asr,
+  ]
+sidebar_position: 2
+---
+
+# Speaker identification
+
+Speaker identification lets you tag speakers consistently across recordings with the help of speaker identifiers. You can generate these string-encoded voice representations using short audio samples of the target speakers.
+
+By tagging known speakers with consistent labels, speaker identification makes transcripts more accurate, searchable, and easier to analyze over time. Providing speaker identifiers can also increase the accuracy of our diarization system.
+
+## Use cases
+
+- **Call centers** – Recognize and tag individual agents and returning customers by name for personalized service, training, and compliance tracking.
+- **Video conferences** – Automatically label participants across multiple meetings to know who said what and maintain consistent speaker records and analytics.
+- **Medical consultations** – Identify doctors and patients across sessions for accurate records and follow-up care.
+- **Media production** – Consistently label recurring speakers or public figures across episodes or segments, which is valuable in subtitling, media search and archiving.
+
+## How it works
+
+To use speaker identification you must enable diarization in one of the two supported modes —
+   [speaker diarization](../features/diarization.mdx#speaker-diarization) or
+   [channel & speaker diarization](../features/diarization.mdx#channel--speaker-diarization) and then follow the two steps below:
+
+- **Enrollment** — For each speaker you want to recognize, generate identifiers from short audio clips (5–30s) where they ideally speak alone.
+  To improve robustness, you can enroll the same speaker with multiple clips recorded under different acoustic conditions, selected to represent the degree of variety and quality that could be expected in the target audio.
+
+- **Identification** — Use the enrolled identifiers in transcription jobs to label known speakers with meaningful names (for example, `Alice` or `John`).  The system matches voices to identifiers and tags the output with the desired labels.
+
+:::info
+Labels for identified speakers must not use reserved internal labels (UU, S1, S2, etc.) and must not include leading or trailing spaces.
+:::
+
+## Known limitations and caveats
```

Comment:
Can we call this something different? Limitations sounds so negative haha

Reply:
used "Known caveats" for now